src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -207,7 +207,8 @@
 
     private ContentAutoCompleters autoCompleters;
 
-    private SearchQuery currentSearchQuery;
+    /** the query the user searches when this basepanel is active */
```

Comment:
Please use `//` as comment for fields. (Or quote some manual where block comments are recommended)